addons/web/static/src/views/view_dialogs/select_create_dialog.js

```diff
@@ -13,7 +13,6 @@ export class SelectCreateDialog extends Component {
         this.state = useState({ resIds: [] });
         this.baseViewProps = {
             display: { searchPanel: false },
-            editable: false, // readonly
```

Comment:
Cannot be inline editable, but shouldn't be readonly (we want to be able to resequence)

Reply:
You do not want to resequence in a select create list view: the only purpose is to select records, not to modify them

Reply:
@aab-odoo Is "Fields > Many2oneField > search more in many2one: resequence inside" a bad test then ?